R/globals.R

```diff
@@ -0,0 +1 @@
+DATA_FETCHING_INTERVAL <- 60 * 3600 * 1000
```

Comment:
This seems too big, don't you mean 60 * 60 * 1000?

Reply:
Good catch. Fixed